app/pages/sections/order-items/catalogue-solutions/delete/contextCreator.js

```diff
@@ -0,0 +1,12 @@
+import manifest from './manifest.json';
+import { baseUrl } from '../../../../../config';
+import { addParamsToManifest } from '../../../../../helpers/contextCreators/addParamsToManifest';
+
+export const getContext = ({ orderId, orderItemId, solutionName, orderDescription }) => {
+  const context = ({
+    ...addParamsToManifest(manifest, {orderId, orderItemId, solutionName }),
+    orderDescription,
+    backLinkHref: `${baseUrl}/organisation/${orderId}/catalogue-solutions/${orderItemId}`,
+  });
+  return context;
+};
```

Comment:
There are some linting issues in this file:

```shell
app/pages/sections/order-items/catalogue-solutions/delete/contextCreator.js
  5:28  error  Expected a line break after this opening brace   object-curly-newline
  5:83  error  Expected a line break before this closing brace  object-curly-newline
  7:38  error  A space is required after '{'                    object-curly-spacing
```

Reply:
resolved